### ISSUE TITLE
unify the key binds for the rings

### ISSUE THREAD

**Issue by blackdragonIV** [sinkillerj/ProjectE#979]
since my old issue got turned down here is a re base of the issue that hopefully will be better

currently trigger buttons for rings powers are different from ring to ring

example swiftwolf mob push ability is activated with right click while harvest goddess band is activated with G also the same goes for arcana ring

please unify all rings to have correct key binds so one will not get confused and burn down their house, if all binds can be done in the same way as back in EE2 it would be great

**Comment by williewillus** [comment]
Guess what? 
in EE2 they were also a mess

**Comment by blackdragonIV** [comment]
back in ee2 it just made sense how the binding worked i mean right click fires projectile a quick access to a mouse click  useful in pvp case... anyway at least can you look into unifying the binds so not every ring bind is  up in the air, though you might see it as a mess of a bind back in ee2 but the original author might have made it a mess for the sake of easier to find button or just something you know where to look for and press while on the go, in any case i think it is i am just used to these buttons and i can get used to a new setup of binds at least all the rings will be in sync

**Comment by blackdragonIV** [comment]
i guess this might be more of a code to write but how about a config that gives player the option to have the old messy ee2 key binds or the new whatever set of key binds for the rings will have

**Comment by MaPePeR** [comment]
we are not adding a config option to trigger between messy keybindings.... 

I agree with unifying: but that means that all instances of PE will be unified - not only some of them. (Thats the whole point of unifying them: to be the same)

**Comment by blackdragonIV** [comment]
@MaPePeR fair enough unifying is the target of this ticket after all

**Comment by sinkillerj** [comment]
If we feel like changing keybinds in the future, we will, until then I ask that duplicate issues are not created.

**Comment by williewillus** [comment]
Tldr = This will be looked at at some point in the not-near future. It will not be changed on its own, but probably will be part of a larger refact of the items. Thanks for the patience.
